src/soc/ibm/power9/Makefile.inc

```diff
@@ -5,6 +5,9 @@ ifeq ($(CPU_IBM_POWER9),y)
 bootblock-y += bootblock.c
 postcar-y += exit_car.S
 romstage-y += romstage.c
+romstage-y += istep_8_9.c
+romstage-y += istep_8_10.c
+romstage-y += ../../../cpu/power/power9/scom.c
```

Comment:
This file does not exist, it was moved to `cpu/power9/scom.c`, but even then it already is in different Makefile.inc, selected by `select CPU_POWER9` in mainboard's Kconfig.

Reply:
Removed this select